docker-compose.yml

```diff
@@ -6,20 +6,38 @@ services:
       context: .
       dockerfile: Dockerfile
     command: ["gunicorn", "-b", "0.0.0.0:5000", "-w", "5", "conbench:application", "--access-logfile=-", "--error-logfile=-", "--preload"]
+    depends_on:
+      - dex
+      - db
     environment:
       APPLICATION_NAME: "Conbench"
       DB_USERNAME: "postgres"
+      # From the docker-compose docs: "By default Compose sets up a single
+      # network for your app. Each container for a service joins the default
+      # network and is both reachable by other containers on that network, and
+      # discoverable by them at a hostname identical to the container name."
+      # That is why the DNS name `db` works.
       DB_HOST: "db"
       DB_PASSWORD: "postgres"
       DB_NAME: "postgres"
       DB_PORT: "5432"
       FLASK_APP: "conbench"
       FLASK_ENV: "development"
+      FLASK_DEBUG: "true"
       REGISTRATION_KEY: "code"
       SECRET_KEY: "Person, woman, man, camera, TV"
-    depends_on:
-      db:
-        condition: service_healthy
+      OAUTHLIB_INSECURE_TRANSPORT: 1
+      GOOGLE_CLIENT_ID: conbench-test-client
+      GOOGLE_CLIENT_SECRET: AnotherStaticSecret
+      CONBENCH_LOG_LEVEL_STDERR: DEBUG
```

Comment:
CC @austin3dickey that's actually how I after all set the log level to DEBUG in my case. Either I remove this here again before merging this PR, or we leave this in here for everyone. Are we sure `docker-compose.yml` is only ever used for dev/CI purposes?